### PR TITLE
Clean up: string property

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/research/stringProperty.schema.tpl.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
     "value": {
       "type": "string",
-      "_instruction": "Enter a text value for this property."
+      "_instruction": "Enter the text value that is described by this string property."
```
Comment (lzehl) [inline schemas/research/stringProperty.schema.tpl.json:14]: that seems wrong.... "Enter the text value of this string property."

**Review by lzehl — COMMENTED** [review]
@apdavison or @Peyman-N I need your input here

File: schemas/research/stringProperty.schema.tpl.json (hunk 1 above)
Comment (Peyman-N) [reply schemas/research/stringProperty.schema.tpl.json:14]: No I think it is correct, but I am not sure.

**Review by Peyman-N — APPROVED** [review]

File: schemas/research/stringProperty.schema.tpl.json (hunk 1 above)
Comment (Peyman-N) [reply schemas/research/stringProperty.schema.tpl.json:14]: Either choice seems good enough, so I'm happy to approve it no matter which one you go with.

Comment (lzehl) [reply schemas/research/stringProperty.schema.tpl.json:14]: for me "is described by" means that this is indirect (I'm describing something...) but here the text value itself is the entity. But maybe it's just me. leave it if you prefer it

**Review by lzehl — APPROVED** [review]

**Pull request merged**